crates/pop-contracts/src/build.rs

```diff
@@ -1,12 +1,20 @@
 // SPDX-License-Identifier: GPL-3.0
 
+use crate::utils::helpers::get_manifest_path;
 use contract_build::{execute, BuildMode, ExecuteArgs};
-use std::path::PathBuf;
+use std::path::{Path, PathBuf};
+use thiserror::Error;
 
-use crate::utils::helpers::get_manifest_path;
+#[derive(Error, Debug)]
+pub enum Error {
```

Comment:
Is cleaner to keep the list of errors used in this file here instead of the errors.rs file: https://github.com/r0gue-io/pop-cli/pull/222/files#diff-3dd0c55b6cdcc6839cb0e4ff1f593a4335a4c8d2b7132a32d450f47cdb30285b.

Reply:
Its tricky. 

By having an error per module, the resulting error perhaps gives greater context as to the source of the error - e.g. Error::BuildError(build::Error::ManifestError(..)) but then there are also circumstances where there might be repetitive errors that are defined in the base Error already.

It just feels like the main Error is becoming a dumping ground for random error types, but maybe that is okay.

Reply:
Note that some of this has been reorganised by moving manifest loading to the common crate.